### PR TITLE
Checklist for drafting postmortems

### PR DESCRIPTION
**Pull request by brittag**
Added a list of suggestions to help ourselves write user-centered postmortems, based on [this draft](https://docs.google.com/document/d/1QsWhclUqECxeR2D7XFuifXmBs7kmOsAgmRRSakmflUo/edit#heading=h.6oqzwohjvprr). Asked for feedback on the set of questions from a couple random cloud.gov-using 18F team members in the office and integrated their thoughts. :)

### CODE REVIEW

File: content/docs/ops/service-disruption-guide.md [hunk 1]
```diff
@@ -62,3 +62,23 @@ A postmortem is not necessary to close, and it does not have to happen immediate
 The person who closes the event should then put a card on the component-owning squad’s board in the urgent lane about writing and posting a postmortem.
 
 For non-security-sensitive work, work to resolve root causes does not have to be completely done before we post the postmortem.
+
+### Checklist for drafting postmortems
+
+We typically use a **What happened** + **What we're doing** structure.
+
+Try checking your draft against the following questions that customers typically have:
```
Comment (khandelwal) [inline content/docs/ops/service-disruption-guide.md]: Perhaps be more assertive: Check your draft ...

File: content/docs/ops/service-disruption-guide.md [hunk 2]
```diff
@@ -62,3 +62,23 @@ A postmortem is not necessary to close, and it does not have to happen immediate
...
+
+* What part of the system had a problem, in terms that I recognize?
```
Comment (khandelwal) [inline content/docs/ops/service-disruption-guide.md]: part(s)

File: content/docs/ops/service-disruption-guide.md [hunk 3]
```diff
@@ -62,3 +62,23 @@ A postmortem is not necessary to close, and it does not have to happen immediate
...
+* How long did the problem last?
+* What was the effect on my applications and my own users?
```
Comment (khandelwal) [inline content/docs/ops/service-disruption-guide.md]: Can we highlight this as important? We need to clearly answer what user-facing application downtime was (if any).

**Review by khandelwal — DISMISSED** [review]

**Comment by khandelwal** [comment]
Approved, because I like this in principle, but provided some comments in case you want do a quick iteration.

**Review by mogul — APPROVED** [review]

**Pull request merged**